Sources/HaCWebsite/main.swift

```diff
@@ -1,3 +1,15 @@
 import HaCWebsiteLib
+import DotEnv
+import Foundation
 
-serveWebsite()
+DotEnv.load()
+
+// Check for required env vars
+guard let _ = DotEnv.get("DATA_DIR") else {
+  fatalError("Missing environment variable")
```

Comment:
We can still fail hard and fast even when the check is colocated with the code that uses it - just needs to be done statically. 

Alternatively, we could make it optional (to make getting started easier) and make a note on `/workshops` when the repository is not loaded. This isn't necessarily the best example, but every developer is not going to need every feature of our website.